etcd/etcd-cluster-crd.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  name: etcdclusters.etcd.database.coreos.com
+spec:
+  conversion:
+    strategy: None
+  group: etcd.database.coreos.com
+  names:
+    kind: EtcdCluster
+    listKind: EtcdClusterList
+    plural: etcdclusters
+    shortNames:
+    - etcd
+    singular: etcdcluster
+  scope: Namespaced
+  version: v1beta2
+  versions:
+  - name: v1beta2
+    served: true
+    storage: true
```

Comment:
is this to avoid the error on the first `apply` saying the type doesn't exist?

Reply:
Yes, on a new cluster in CI, it really simplifies things if you only need to apply all manifests once.